app/views/citizens/consents/show.html.erb

```diff
@@ -11,33 +11,23 @@
         <p class="govuk-body govuk-!-padding-bottom-2"><%= t(".inset_text") %></p>
       <% end %>
 
-      <details class="govuk-details govuk-!-margin-top-4" data-module="govuk-details">
-        <summary class="govuk-details__summary">
-          <span class="govuk-details__summary-text"
-            title="<%= t(".summary_heading") %>"
-            aria-label="<%= t(".summary_heading") %>">
-            <%= t(".summary_heading") %>
-          </span>
-        </summary>
-
-        <div class="govuk-details__text">
-          <p><%= t(".able_to_see.heading") %></p>
-
-          <ul class="govuk-list govuk-list--bullet">
-            <% t(".able_to_see.list").each_line do |item| %>
-              <li><%= item %></li>
-            <% end %>
-          </ul>
-
-          <p><%= t(".unable_to_see.heading") %></p>
-
-          <ul class="govuk-list govuk-list--bullet">
-            <% t(".unable_to_see.list").each_line do |item| %>
-              <li><%= item %></li>
-            <% end %>
-          </ul>
-        </div>
-      </details>
+      <%= govuk_details(summary_text: t(".summary_heading")) do %>
+        <p><%= t(".able_to_see.heading") %></p>
```

Comment:
Very minor - and it might not make any difference - but you could stick the `govuk-body` class into a paragraph. A quick scan of the codebase makes it look there's already a mix so it's entirely up to you